src/internal/minikubetestenv/deploy.go

```diff
@@ -878,13 +880,50 @@ func putRelease(t testing.TB, ctx context.Context, namespace string, kubeClient
 	previousOptsHash := getOptsConfigMapData(t, ctx, kubeClient, namespace)
 	pachdExists, err := checkForPachd(t, ctx, kubeClient, namespace, opts.EnterpriseServer)
 	require.NoError(t, err)
-	if mustUpgrade {
+	if mustUpgrade || opts.InstallPrometheus {
 		t.Logf("Test must upgrade cluster in place, upgrading cluster with new opts in %v", namespace)
 		require.NoErrorWithinTRetry(t,
 			time.Minute,
 			func() error {
 				return errors.EnsureStack(helm.UpgradeE(t, helmOpts, chartPath, namespace))
 			})
+		// get latest version with: helm pull prometheus-community/kube-prometheus-stack -d etc/helm/charts/
+		require.NoErrorWithinTRetry(t, time.Minute, func() error {
+			chartPath := localPath(t, "etc", "helm", "charts", "kube-prometheus-stack-55.7.1.tgz")
```

Comment:
I see that 55.11.0 is the latest version.

Reply:
Wow, that’s several versions in six days! I’ll update.